vendor/assets/javascripts/initializers.js

```diff
@@ -54,4 +54,23 @@ $(document).on('turbolinks:load', function() {
       }
     });
   });
+  $('.js-dropzone-featured-image').each(function() {
+    $(this).find('form').dropzone({
+      dictDefaulMessage: 'Drop files or click here to upload',
+      init: function() {
+        this.on('success', function(req, res) {
+          var clone = $($.clone($('.featured-image-table-element')[0]));
```

Comment:
make sure you are cloning on the instance of ```.js-featured-image-table-element```. This would get teh first one no matter where it is on the page so you couldn't have two. Traverse from the form element to get it rather than just declare the class